### DISCUSSION TITLE
Changing mode from balance to silent but the cpu fan is still running

### DISCUSSION THREAD

**Discussion by thduc87** [seerge/g-helper#358] [General]
Hi, i'm using g-helper 0.58 and just run into a weird situation today when switching performance mode from balance to silent but the cpu fan is still running

**Comment by seerge** [comment]
@thduc87 silent mode is built in your bios, and still has own fan curve. How fans will work really depends on your model / current temps

**Reply by thduc87** [reply]
my model is g14 2022, R7 6800HS, RX6700S, 16Gb, the issue i reported is not something serious but i find out after several times go back and forth between modes, in the end all fans did go back to 0rpm after go back to balance then switch back to silent again
